docs/source/guide/explanation/algorithms/object_detection/object_detection.rst

```diff
@@ -1,17 +1,116 @@
+
+The common approach to building object detection architecture is to take a feature extractor (backbone), that can be inherited from the classification task.
+Then goes a head that calculates coordinates and class probabilities based on aggregated information from the image.
+Additionally, some architectures use `Feature Pyramid Network (FPN) <https://arxiv.org/abs/1612.03144>`_ to transfer and process feature maps from backbone to head and called neck.
+
+For the supervised training we use the following algorithms components:
+
+.. _od_supervised_pipeline:
+
+- ``Augmentations``: We use random crop and random rotate, simple bright and color distortions and multiscale training for the training pipeline.
+
+- ``Optimizer``: We use `SGD <https://en.wikipedia.org/wiki/Stochastic_gradient_descent>`_ optimizer with the weight decay set to **1e-4** and momentum set to **0.9**.
+
+- ``Learning rate schedule``: `ReduceLROnPlateau <https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html>`_. This learning rate scheduler proved its efficiency in dataset-agnostic trainings, its logic is to drop LR after some time without improving the target accuracy metric. Also, we update it with ``iteration_patience`` parameter that ensures that a certain number of training iterations (steps through the dataset) were passed before dropping LR.
+
+- ``Loss function``: We use `Generalized IoU Loss <https://giou.stanford.edu/>`_  for localization loss to train the ability of the model to find the coordinates of the objects. For the classification head, we use a standard `FocalLoss <https://arxiv.org/abs/1708.02002>`_.
+
+- ``Additional training techniques``
+    - ``Early stopping``: To add adaptability to the training pipeline and prevent overfitting. You can use early stopping like the below command.
+      
+      .. code-block::
+
+        $ otx train {TEMPLATE} ... \
+            params \
+            --learning_parameters.enable_early_stopping=True
+
+    - `Anchor clustering for SSD <https://arxiv.org/abs/2211.17170>`_: This model highly relies on predefined anchor boxes hyperparameter that impacts the size of objects, which can be detected. So before training, we collect object statistics within dataset, cluster them and modify anchor boxes sizes to fit the most for objects the model is going to detect. 
+    
+    - ``Backbone pretraining``: we pretrained MobileNetV2 backbone on large `ImageNet21k <https://github.com/Alibaba-MIIL/ImageNet21K>`_ dataset to improve feature extractor and learn better and faster.
+
+
 **************
 Dataset Format
 **************
 
-To be added soon
+At the current point we support `COCO <https://cocodataset.org/#format-data>`_, 
+`Pascal-VOC <https://openvinotoolkit.github.io/datumaro/docs/formats/pascal_voc/>`_ and
+`YOLO <https://openvinotoolkit.github.io/datumaro/docs/formats/yolo/>`_ dataset format.
+The dataset should be formatted according to the structure below:
+
+.. code::
```

Comment:
Is this structure same for all dataset format(including VOC, YOLO)? If not, this part should be added a explanation that this is example for COCO dataset format.

Reply:
Nice comment @jaegukhyun , we should differentiate the dataset format

Reply:
Thank you for this comments, I mentioned COCO format